plugins/webhook-egress/src/webhook.egress.spec.ts

```diff
@@ -0,0 +1,97 @@
+import api from '@flatfile/api'
+import { FlatfileEvent } from '@flatfile/listener'
+import {
+  createRecords,
+  deleteSpace,
+  setupListener,
+  setupSimpleWorkbook,
+  setupSpace,
+} from '@flatfile/utils-testing'
+import { webhookEgress } from './webhook.egress'
+
+jest.setTimeout(10_000)
+
+describe('webhookEgress() e2e', () => {
+  const listener = setupListener()
+
+  let spaceId
+  let workbookId
+  let sheetId
+
+  beforeAll(async () => {
+    const space = await setupSpace()
+    spaceId = space.id
+    const workbook = await setupSimpleWorkbook(space.id, [
+      'name',
+      'email',
+      'notes',
+    ])
+    workbookId = workbook.id
+    sheetId = workbook.sheets[0].id
+    await createRecords(sheetId, [
+      {
+        name: 'John Doe',
+        email: 'john@doe.com',
+        notes: 'foobar',
+      },
+      {
+        name: 'Jane Doe',
+        email: 'jane@doe.com',
+        notes: 'foobar',
+      },
+    ])
+  })
+
+  afterAll(async () => {
+    await deleteSpace(spaceId)
+  })
+
+  it('returns successful outcome message', async () => {
+    const webhookUrl =
+      'https://webhook.site/077ab8ee-8570-48b0-82d7-f7561ae75dc2'
```

Comment:
Can we make this an environment variable so that we can swap it out easily / have different local and prod environments?

Reply:
Resolved with [6ce71f1](https://github.com/FlatFilers/flatfile-plugins/pull/212/commits/6ce71f127cc628f3345df9d44f07112fb548b7e7)